.github/workflows/dependency-update.yaml

```diff
@@ -0,0 +1,26 @@
+name: update-versions-with-renovate
+
+on:
+  push:
+    branches: [main, master]
+  schedule:
+    # https://crontab.guru/
+    # once a day
+    - cron: '0 0 * * *'
+
+jobs:
+  update-versions-with-renovate:
+    runs-on: ubuntu-latest
+    if: github.repository == 'kubernetes-sigs/external-dns'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4.2.2
+      # https://github.com/renovatebot/github-action
+      - name: self-hosted renovate
+        uses: renovatebot/github-action@v41.0.14
+        with:
+          # https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm not sure to see how it can work.

AFAIK, it's supposed to be a RENOVATE_TOKEN, not a GITHUB one.
https://github.com/renovatebot/github-action

Github action can be tested with [act](https://github.com/nektos/act).